tests/helpers.py

```diff
@@ -50,9 +50,11 @@ def fixture_params(fixture_name, params):
 
```

Comment:
Changed this to be setable by individual tests so we can change it depending on what test data we're using